extensions/3DTILES_bounding_volume_S2/0.0.0/schema/extension.schema.json

```diff
@@ -0,0 +1,35 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema",
+  "title": "3DTILES_bounding_volume_S2 extension",
+  "type": "object",
+  "properties" : {
+    "s2cell": {
+      "type": "object",
+      "properties": {
+        "token": {
+          "type": "string",
+          "description": "A hexadecimal representation of the S2CellId. Tokens must be lower-case (except X), must not contain whitespace, must have trailing zeros stripped. The 0 S2CellId must be represented as 'X'."
```

Comment:
Also consider moving the sentence about the meaning of `X` before the sentence about case requirements.